### ISSUE TITLE
Enhancement request: Higher resolution output

### ISSUE THREAD

**Issue by kamakazikamikaze** [MJL85/mnet#6]
I understand that there may be limitations to what the Python modules for the image generation can handle but I'd still like to request that .PNG files have a higher resolution. Devices and connections can be barely legible when zoomed in.

It appears that the output image has a resolution of 96ppi. Could this be raised _or_ have the option of a .SVG file for output? SVG would be lightweight and can be scaled without losing quality.

**Comment by MJL85** [comment]
Actually it _can_ output svg!  Which is pretty sweet.  I noticed graphviz messes up some of the text alignment, but like you said it can scale better than png.  Just use an output file with the extension .svg instead of .png to product it.  Hope this helps.  

I may also include a command line option to change the size of the text.

**Comment by kamakazikamikaze** [comment]
Oh, thanks! I know it was mentioned somewhere (Reddit maybe?) but the README.md didn't include it as an option. I'll try it out

**EDIT**

Works perfectly! Expanding the text or allowing a selection from a list of pre-defined values for context/legibility would certainly help. This is great!

**Comment by MJL85** [comment]
Cool!  I'll add an option to change the default point size.  I should have done that before but I was being lazy.

**Comment by kamakazikamikaze** [comment]
I'd say "occupied" instead of "lazy" given the changes you've had to make recently.

Just as a side note: it would make it much neater and easier to follow if the node levels were given more depth too. [Some lines are hard to follow with larger networks](https://drive.google.com/file/d/0B1qZrP_GY8KXTjQxUUFpWk4xSEk/view?usp=sharing). All in due time I suppose!

**Comment by MJL85** [comment]
That's a pretty wide network.  Hmm..  

I noticed a lot of links with specific VLANs with only one downstream device.  Are those phones maybe?  You can use the config file to exclude the subnet associated with VLAN 505 if that information is irrelevant to you.

**Comment by kamakazikamikaze** [comment]
Shoot, I hadn't considered setting them in an exclude. The challenge (for me) is that we use 10.0.0.0 thru 10.68.255.0 (roughly 3392 VLANs in all). However, because our equipment is on a more limited range of VLANs I suppose I should just specify each of them individually under the "subnets" section? 

**EDIT**

I was able to filter them out easily, much easier than I imagined.
You can go ahead and close this ticket. With .SVG as an optional output format it's enough to meet my needs. The downfall of a network our size is that only Visio can open the file (it's too wide for GIMP or Adobe Illustrator) but even at that we can zoom in and out with ease. Not only that, textual output isn't covered by any lines so there's no problem with data being shoved to the back.

I don't want you to have to calculate some proportion that would expand and drop the entities when the breadth of the network is high due to how many tries you'd have to do to 'perfect' it. If push comes to shove I can try tweaking Illustrator to open wide images and from there manually change the rows to drop down more.
